### PR TITLE
Case sensitivity cleanup

### PR DESCRIPTION
**Pull request by peterp**
A quick cleanup, something more focused is required. Subsequent pull-requests incoming.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  buttonText: {
+    color: '#fff',
+    fontFamily: 'Brandon Text',
+    fontSize: 13,
+    fontWeight: 'bold',
+    marginTop: 15,
+    marginBottom: 17,
+    letterSpacing: 1,
+  },
+});
```
Comment (justinvdm) [inline src/components/Button.js]: Are these styles moved from elsewhere in the codebase? Just wondering if there needs to be a corresponding deletion.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  button: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#f77040',
+    borderRadius: 5,
+    marginLeft: 16,
+    marginRight: 16,
+  },
+  buttonIsDisabled: {
+    backgroundColor: '#dfe5e6',
```
Comment (justinvdm) [inline src/components/Button.js]: Maybe not for this PR, but wondering if we should alias our colours as constants in some style vars module.

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  setNativeProps(nativeProps) {
+    this._root.setNativeProps(nativeProps);
+  }
+
+  render() {
+    let button = (
+      <View
+        style={[
+          styles.button,
+          this.props.disabled ? styles.buttonIsDisabled : undefined,
```
Comment (justinvdm) [inline src/components/Button.js]: [A bit nitpicky](https://shapeshed.com/the-void-of-undefined-in-javascript), but wondering if we could use `null` here instead (provided React will handle it the same way as `undefined`)?

**Comment by justinvdm** [comment]
:+1: looks good :) left questions, but I don't think they should prevent this from landing.

File: src/components/Button.js (hunk 1 above)
Comment (peterp) [inline src/components/Button.js]: They were in `./src/StyleSheet.js,` I will eventually be moving everything out of there.

File: src/components/Button.js (hunk 2 above)
Comment (peterp) [inline src/components/Button.js]: 👍

File: src/components/Button.js (hunk 3 above)
Comment (peterp) [inline src/components/Button.js]: 👍

**Pull request merged**